### ISSUE TITLE
`can't compare offset-naive and offset-aware datetimes` when getting expired

### ISSUE THREAD

**Issue by whwright** [libkeepass/pykeepass#115]
Here is the example code, with my values redacted:

```python
>>> from pykeepass import PyKeePass
>>> kp = PyKeePass('REDACTED', password='REDACTED')
>>> for entry in kp.find_entries(title='entry that has an expiration'):
...     print(entry.expired)
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/home/whw/.virtualenvs/kpcat/lib/python3.5/site-packages/pykeepass/baseelement.py", line 153, in expired
    return self.expires and (datetime.utcnow() > self.expiry_time)
TypeError: can't compare offset-naive and offset-aware datetimes
```

Upon further inspection it appears that `self.expiry_time` has `tzinfo=UTC` and `datetime.utcnow()` does not!

**Comment by whwright** [comment]
I have a fix for this, I am just trying to figure out where to commit it since there are commits between 3.0.2 and master.

**Comment by Toffl** [comment]
Did you commit the fix somewhere? I ran into the same error today

**Comment by whwright** [comment]
Fix is in PR #116

**Comment by Evidlo** [comment]
Can be closed
